src/Debugger/Daemon.php

```diff
@@ -155,6 +155,23 @@ private function setBreakpoints($breakpoints)
     }
 
     private function defaultUniquifier()
+    {
+        $dir = new \RecursiveDirectoryIterator($this->sourceRoot);
+        $iterator = new \RecursiveIteratorIterator($dir);
+        $regex = new \RegexIterator(
+            $iterator,
+            '/^.+\.php$/i',
+            \RecursiveRegexIterator::GET_MATCH
+        );
+
+        $files = array_keys(iterator_to_array($regex));
```

Comment:
When will this method be called?
I'm interested in how long it takes for somewhat larger app with lots of dependencies in the vendor dir.

Reply:
This will be called in the daemon's process (so not in the request cycle). Yes, it's expensive as it's looking at all the source files but it will only be called once if there is no uniquifier specified.

Reply:
ok